build.cmd

```diff
@@ -242,12 +242,10 @@ set __RunArgs=-BuildOS=%__BuildOS% -BuildType=%__BuildType% -BuildArch=%__BuildA
 
 if %__EnforcePgo%==1 (
     if %__BuildArchArm%==1 (
-        echo Error: enforcepgo cannot be used with arm architecture
-        goto Usage
+        echo WARNING: enforcepgo does nothing with arm architecture
```

Comment:
It just says warning. I can change it to INFO, or whatever else to avoid confusion. I just don't want it to silently do nothing